File: dataset_reduzido/avatar/Python/Code/codeforces_519_B.py
n = int ( input ( ) )
a = list ( map ( int , input ( ).split ( ) ) )
b = list ( map ( int , input ( ).split ( ) ) )
c = list ( map ( int , input ( ).split ( ) ) )
x = 0
y = 0
z = 0
for i in a :
    x += i
for i in b :
    y += i
for i in c :
    z += i
print ( abs ( x - y ) )
print ( abs ( y - z ) )

File: output/Qwen_repaired_1/avatar/Python/Java/codeforces_519_B.java
import java.util.Scanner;

public class codeforces_519_B {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = Integer.parseInt(scanner.nextLine());
        String[] splitA = scanner.nextLine().split(" ");
        String[] splitB = scanner.nextLine().split(" ");
        String[] splitC = scanner.nextLine().split(" ");

        int x = 0;
        int y = 0;
        int z = 0;

        for (String s : splitA) {
            x += Integer.parseInt(s);
        }

        for (String s : splitB) {
            y += Integer.parseInt(s);
        }

        for (String s : splitC) {
            z += Integer.parseInt(s);
        }

        System.out.println(Math.abs(x - y));
        System.out.println(Math.abs(y - z));
        scanner.close();
    }
}